torchdet3d/utils/utils.py

```diff
@@ -115,25 +109,42 @@ def load_checkpoint(fpath):
         raise
     return checkpoint
 
-def load_pretrained_weights(model, file_path='', pretrained_dict=None, resume=False, parallel_model=False):
+def _print_loading_weights_inconsistencies(discarded_layers, unmatched_layers):
+    if discarded_layers:
+        print(
+            '** The following layers are discarded '
+            'due to unmatched keys or layer size: {}'.
+            format(pformat(discarded_layers))
+        )
+    if unmatched_layers:
+        print(
+            '** The following layers were not loaded from checkpoint: {}'.
+            format(pformat(unmatched_layers))
+        )
+
+def load_pretrained_weights(model, file_path='', pretrained_dict=None):
     r"""Loads pretrianed weights to model. Imported from openvinotoolkit/deep-object-reid.
     Features::
         - Incompatible layers (unmatched in name or size) will be ignored.
         - Can automatically deal with keys containing "module.".
     Args:
         model (nn.Module): network model.
         file_path (str): path to pretrained weights.
-    Examples::
-        >>> from torchreid.utils import load_pretrained_weights
-        >>> file_path = 'log/my_model/model-best.pth.tar'
-        >>> load_pretrained_weights(model, file_path)
     """
+    def _remove_prefix(key, prefix):
+        prefix = prefix + '.'
+        if key.startswith(prefix):
+            key = key[len(prefix):]
+        return key
+
     if file_path:
         check_isfile(file_path)
     checkpoint = (load_checkpoint(file_path)
                        if not pretrained_dict
                        else pretrained_dict)
 
+    if 'classes_map' in checkpoint:
```

Comment:
Where do you use it?

Reply:
sorry, I've decided to update this function from deep-person-reid with the latest functionality of printing from LB, but have forgottenten to delete this